### PR TITLE
 instructor: Results display format can be improved for questions with no recipient #2844 

### LINKED ISSUE #2844: Instructor: view results: improve display format for questions with no specific recipient
When general is the recipient, recipient name and team name are "-".

untitled

The following pic is even more confusing for "- Received Response.." and "- Detailed Response"
untitled2

### PR DESCRIPTION
**Pull request by yj-soh**
Fixes #2844

Shouldn't have any side effects, the methods affected are:
`getTeamNameFromRoster()` - Getter used in csv string builder for recipients in `FeedbackSessionsLogic`.
`getTeamNameForEmail()` - Getter used in `Feedback*QuestionDetails` classes.

"[General Feedback]" should suggest sufficiently that it is a question targeted at nobody specifically while being an impossible team name (as it starts with a non-alphanumeric character). [This mockup](http://codepen.io/yj-soh/full/ZYZLpg) can be used to test potential replacement strings.

general-feedback

The replacement string is still tentative, but the following tests will have to be revised upon confirmation, before merging:
> logic.FeedbackSessionsLogictest
> ui.browsertests.InstructorFeedbackResultsPageUiTest
> ui.browsertests.StudentFeedbackResultsPageUiTest
> ui.browsertests.AdminHomePageUiTest

All else is dev green.

### CODE REVIEW

**Comment by kanghj** [comment]
For the heading, maybe rephrase to "Not for a specific recipient" for the outermost heading, then try to make it blank for the top of the panel body? 

heading
heading2


Maybe it's better to just leave the team name and recipient name as "-", it seems more meaningful that way.

**Comment by yj-soh** [comment]
Handling those individually will require changing the jsp files. Shouldn't be much of an issue for extensibility in this case, though, since only the views first grouped by Recipient need to be changed.
I updated the mockup to include the option for that design.

As for leaving it as "-", I think only the *Received Responses Statistics* and *Detailed Responses* subheadings (in the RQG view) are of any problem since the extra "*-*" doesn't help. The subsection heading can definitely benefit from a *Not for a specific recipient* instead of "*-*", but it's not that significant in my opinion.
